torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -176,12 +160,12 @@ The explanation needs to be clear on why this is needed. Here are some good exam
 
     if (!rejection_reason) {
       await this.logger.log("merge", extra_data);
-      await this.dispatchEvent(
-        "try-merge",
-        forceRequested,
-        mergeOnGreen,
-        landChecks || landChecksEnrolled
-      );
+      await this.dispatchEvent("try-merge", {
```

Comment:
It's a nice tweak indeed to use the payload dictionary

Reply:
yea its pretty much https://github.com/pytorch/test-infra/pull/377 but i think that pr is a bit abandoned